sw/src/service-worker/service-worker.js

```diff
@@ -0,0 +1,395 @@
+import { clientsClaim } from 'workbox-core'
+import { precacheAndRoute, createHandlerBoundToURL } from 'workbox-precaching'
+import { registerRoute, setDefaultHandler } from 'workbox-routing'
+import {
+    NetworkFirst,
+    StaleWhileRevalidate,
+    Strategy,
+} from 'workbox-strategies'
+import {
+    openSectionsDB,
+    deleteSectionsDB,
+    SECTIONS_STORE,
+} from '../lib/sections-db'
+
+function setUpKillSwitchServiceWorker() {
+    // A simple, no-op service worker that takes immediate control and tears
+    // everything down. Has no fetch handler.
+    self.addEventListener('install', () => {
+        self.skipWaiting()
+    })
+
+    self.addEventListener('activate', async () => {
+        // Unregister, in case app doesn't
+        self.registration.unregister()
+        // Delete all caches
+        const keys = await self.caches.keys()
+        await Promise.all(keys.map(key => self.caches.delete(key)))
+        // Delete DB
+        await deleteSectionsDB()
+        // Force refresh all windows
+        const clients = await self.clients.matchAll({ type: 'window' })
+        clients.forEach(client => client.navigate(client.url))
+    })
+}
+
```

Comment:
See the below function's usage in the shell's [`service-worker.js`](shell/src/service-worker.js)